JupyterHub/Vagrantfile

```diff
@@ -18,42 +18,55 @@ Vagrant.configure("2") do |config|
         node.vm.hostname = "jupyterhub"
         node.disksize.size = "50GB"
         node.vm.synced_folder "Shared", "/vagrant_shared"
-        node.vm.synced_folder "../common", "/vagrant_common"
         node.vm.network "public_network"
         node.vm.network "private_network", ip: "10.0.0.40", virtualbox__intnet: true
         node.vm.provision :shell, inline: <<-SHELL
         sudo su
         cd /root
 
         yum -y install https://centos7.iuscommunity.org/ius-release.rpm
-        yum -y install epel-release python36u python36u-dev python36u-devel npm wget net-tools krb5-libs krb5-workstation git krb5-devel
+        yum -y install epel-release python36u python36u-dev python36u-devel npm wget net-tools krb5-libs krb5-workstation git krb5-devel java-1.8.0-openjdk nginx

```

Comment:
No Miniconda?

Reply:
That's something I'd like to do, but I'm trying to stick to commit early commit often. Feel free to file an issue for this, or I'll do it if you don't want to.